src/services/types.ts

```diff
@@ -0,0 +1,10 @@
+export type UserType = {
```

Comment:
Maybe you can create a `src/types` folder and create files for the different types

Reply:
Also I have a few comments about this `UserType`:
* This looks more like an interface than a type, you can read more about this [here](https://blog.logrocket.com/types-vs-interfaces-in-typescript/)

* You could rewrite this like: 
```
type User = {
  pk: number;
  email: string;
  first_name: string;
  last_name: string;
}

interface UserState {
  access_token: string | null;
  refresh_token: string | null;
  user: UserType | null;
}
```
This way you can reuse the User type somewhere else